aiida/orm/importexport.py

```diff
@@ -1162,12 +1168,27 @@ def import_data_dj(in_path, user_group=None, ignore_unknown_nodes=False,
             if pks_for_group:
                 # If user specified a group, import all things in it
                 if user_group:
-                    group = user_group[0]
+                    if isinstance(user_group, str):
+                        # Will try to retrieve Group with provided label
+                        res_group = Group.objects.find(label=user_group)
```

Comment:
This pattern should be replaced with `Group.objects.get_or_create`

Reply:
done